packages/block-editor/src/components/block-pattern-setup/setup-toolbar.js

```diff
@@ -87,10 +83,7 @@ const SetupToolbar = ( {
 			) }
 			{ displayControls }
 			{ isCarouselView && (
-				<Actions
-					onBlockPatternSelect={ onBlockPatternSelect }
-					onStartBlank={ onStartBlank }
-				/>
+				<Actions onBlockPatternSelect={ onBlockPatternSelect } />
```

Comment:
Unrelated to this PR, do you think we should inline the `displayControls` and `Actions` components here? As far as I can tell, they aren't used anywhere else.

Reply:
I guess we can, but personally I don't see much of a difference now. This component is experimental and might go through more design iterations(like the previous one resulting to this PR), so we can look into it then. It doesn't have any performance impact or anything.

Reply:
> This component is experimental and might go through more design iterations(like the previous one resulting to this PR), so we can look into it then.

Sounds good 👍